backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ProfileClient.cs

```diff
@@ -82,5 +82,41 @@ public async Task<UserProfile> GetUserProfile(int userId)
 
             return null;
         }
+
+        /// <inheritdoc/>
+        public async Task<UserProfile> GetUserProfileByUUID(Guid uuid)
```

Comment:
Maybe this method and GetUserProfile can be merged together because it seems like the only thing that's different is the enspointUrl.

Reply:
I think the difference in endpoint is a reason why they should be different methods actually. At least, that's the standard I've seen everywhere else, and a client should after all only do the call to the enpoint (possibly some response handling) and little else. Also, these two have differently typed arguments so they canæt really be the same method.

That being said, I should probably extract the similar code that the two share into a privat method to avoid as much duplicate code as possible.

Reply:
Yeah what I meant was to put a little effort in to better the situation with the duplicate code since it doesn't seem like any effort has been put in to do so.